portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,15 +35,29 @@
   private static final String TASK_NAME = "Comments";
   private static final String TIMESTAMP = "timestamp";
   private static final String COMMENT_PROPERTY = "comment";
+  private static final String FILTER_PARAM = "filterCount";
```

Comment:
Can you rename this servlet to "comments"  (line 33) rather than data? We should be specific in how we name the api

Reply:
Would it also be better to switch from '/data' to something like '/comments'?

Reply:
"/comments"